Makefile

```diff
@@ -35,25 +35,38 @@ help:
 list:
 	@echo ${CITIES}
 
+$(filter %,$(CITIES)): %: \
+  ${DATA_DIR}/%.bbox \
+  ${DATA_DIR}/%.osm.pbf \
+  ${DATA_DIR}/%.gtfs.tar \
+  ${DATA_DIR}/%.nominatim.sql ${DATA_DIR}/%.nominatim_tokenizer.tgz \
+  ${DATA_DIR}/%.photon.tgz \
+  ${DATA_DIR}/%.mbtiles \
+  ${DATA_DIR}/%.graph.obj \
+  ${DATA_DIR}/%.valhalla.tar \
+  tag_images
```

Comment:
nit: visually I usually like multi-line strings with a bunch of backslashes to be indented more than this. It was tricky for me to figure out at first (visually) whether or not this `tag_images` line was a dependancy or part of the target somehow.